controls/overlay.rb

```diff
@@ -137,87 +101,57 @@
-
-    desc 'Only CMS-approved external PKIs have been evaluated to ensure that they have security 
-    controls and identity vetting procedures in place which are sufficient for CMS systems to 
-    rely on the identity asserted in the certificate. PKIs lacking sufficient security controls 
-    and identity vetting procedures risk being compromised and issuing certificates that enable 
-    adversaries to impersonate legitimate users.
+    impact 0.0
+    title 'MongoDB must only accept end entity certificates issued by CMS PKI or CMS-approved PKI Certification Authorities (CAs) for the establishment of all encrypted sessions.'
+    desc 'Only CMS-approved external PKIs have been evaluated to ensure that they have security controls and identity vetting procedures in place which are sufficient for CMS systems to rely on the identity asserted in the certificate. PKIs lacking sufficient security controls and identity vetting procedures risk being compromised and issuing certificates that enable adversaries to impersonate legitimate users.
 
     The authoritative list of CMS-approved PKIs is published at http://iase.disa.mil/pki-pke/interoperability.
 
-    This requirement focuses on communications protection for MongoDB session rather than for 
-    the network packet.'
-
-    tag 'check', 'To run MongoDB in SSL mode, you have to obtain a valid certificate singed 
-    by a single certificate authority. 
+    This requirement focuses on communications protection for MongoDB session rather than for the network packet.'
+    tag 'check', 'To run MongoDB in SSL mode, you have to obtain a valid certificate singed by a single certificate authority. 
 
-    Before starting the MongoDB database in SSL mode, verify that certificate used is issued by 
-    a valid CMS certificate authority (openssl x509 -in <path_to_certificate_pem_file> 
-    -text | grep -i "issuer"). 
+    Before starting the MongoDB database in SSL mode, verify that certificate used is issued by a valid CMS certificate authority (openssl x509 -in <path_to_certificate_pem_file> -text | grep -i "issuer"). 
 
-    If there is any issuer present in the certificate that is not a CMS approved certificate 
-    authority, this is a finding.'
-    
-    tag 'fix', 'Remove any certificate that was not issued by an approved CMS certificate 
-    authority. Contact the organization\'s certificate issuer and request a new certificate 
-    that is issued by a valid CMS certificate authorities.'
+    If there is any issuer present in the certificate that is not a CMS approved certificate authority, this is a finding.'
+    tag 'fix', 'Remove any certificate that was not issued by an approved CMS certificate authority. Contact the organization\'s certificate issuer and request a new certificate that is issued by a valid CMS certificate authorities.'
 
-    desc 'caveat', 'Not applicable for this CMS ARS 3.1 overlay, since the related security 
-    control is not included in CMS ARS 3.1'

```

Comment:
@ejaronne If this control is not applicable anyway, do we need to update the verbiage?